app/projections/audit.py

```
@@ -0,0 +1,16 @@
+"""Audit projection: global event count (all subject types)."""
+
+from __future__ import annotations
+
+from app.application.dtos.event import EventResult
+from app.core.projections import projection
+
+
+@projection("global_event_count", subject_type=None)
+async def global_event_count(state: dict, event: EventResult) -> dict:
```

Ruff (0.15.5)

[warning] Unused function argument: `event`

(ARG001)